app/views/entities/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<p style="color: green"><%= notice %></p>
+<div class="heading">
+  <h1 class="heading__title">Transactions</h1>
+</div>
+<%= link_to "New entity", new_category_entity_path, :class => "fcc-btn" %>
+<div id="entities">
+  <% @entities.each do |entity| %>
```

Comment:
- [ ] Please ensure the current user can see only their own transactions. I sign up with another account and I can still the transactions of another user.